lib/utils.ts

```diff
@@ -412,3 +410,24 @@ export function countOccurrences<T>(collection: Iterable<T>, item: T): number {
     }
     return result;
 }
+
+export function asSafeVer(semver: string | number | null) {
+    if (semver != null) {
```

Comment:
Does this need to be `!==` or is there some subtle JS thing going on here?

Reply:
I don't think undefined _can_ be passed right now, so you're right, it should be strong eq